deploy/crds/apps.openshift.io_servicebindingrequests_crd.yaml

```diff
@@ -251,7 +255,10 @@ spec:
               description: EnvVarPrefix is the prefix for environment variables
               type: string
             mountPathPrefix:
-              description: MountPathPrefix is the prefix for volume mount
+              description: "Important: Run \"operator-sdk generate k8s\" to regenerate
```

Comment:
Better to keep `MountPathPrefix is the prefix for volume mount` description.

Reply:
auto-generated by `operator-sdk generate crd`

Reply:
we would want to remove the `Imortant- ...` note as it is not telling about just the API, but giving additional (unintended) info on generation.

Reply:
Done 25b27f4